pkg/virt-handler/vm.go

```diff
@@ -331,6 +337,12 @@ func (d *VirtualMachineController) clearPodNetworkPhase1(vmi *v1.VirtualMachineI
 	defer d.phase1NetworkSetupCacheLock.Unlock()
 
 	delete(d.phase1NetworkSetupCache, vmi.UID)
+
+	// Delete the VMI Netowrk cache files on Cleanup
+	err := os.RemoveAll(virtutil.VMIInterfaceDir)
```

Comment:
isn't this deleting all cache files? We just want to delete the one associated with the VMI. right?

Reply:
`VMIInterfaceDir` is the VMI's directory named with it's UID
It should delete the one associated with just that VMI.

Reply:
https://github.com/kubevirt/kubevirt/pull/3488/files#diff-6c6a7fefe37261e2fa8a010cf9e7ae08R18